### PR TITLE
Feature 152931 add layers geojson

### PR DESCRIPTION
**Pull request by AlexVinnik**
Add geojson layer

### CODE REVIEW

File: addon/components/layers/geojson-layer.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import BaseLayer from '../base-layer';
+
+/**
+  WFS layer component for leaflet map.
+
+  @class WfsLayerComponent
```
Comment (almostcake) [inline addon/components/layers/geojson-layer.js]: typo

File: addon/components/layers/geojson-layer.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    where 'layer' is metadata of layer related to identification result, features is array
+    containing (GeoJSON feature-objects)[http://geojson.org/geojson-spec.html#feature-objects]
+    or a promise returning such array.
+  */
+  identify(e) {
+    var bounds = new Terraformer.Primitive(e.polygonLayer.toGeoJSON());
+
+    let geojsonLayer = this.get('_leafletObject');
+
+    //console.log("grouplayer: "+ groupLayer);
```
Comment (almostcake) [inline addon/components/layers/geojson-layer.js]: remove

File: addon/layers/geojson.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+*/
+export default BaseLayer.extend({
+  /**
+    Permitted operations related to layer type.
+
+    @property operations
+    @type String[]
+    @default ['edit', 'remove', 'identify', 'search', 'legend']
+  */
+  operations: ['edit', 'remove', 'identify', 'search', 'legend'],
```
Comment (almostcake) [inline addon/layers/geojson.js]: do you actually have legend for this layer?

File: addon/components/layers/geojson-layer.js (hunk 1 above)
Comment (AlexVinnik) [reply addon/components/layers/geojson-layer.js]: переделал в ветке feature-add-markerCluster-geojson

File: addon/components/layers/geojson-layer.js (hunk 2 above)
Comment (AlexVinnik) [reply addon/components/layers/geojson-layer.js]: delete in branche feature-add-markerCluster-geojson

File: addon/layers/geojson.js (hunk 3 above)
Comment (AlexVinnik) [reply addon/layers/geojson.js]: edit in branche "feature-add-markerCluster-geojson"

**Comment by kuzkok** [comment]
Будет в #122

**Pull request closed without merging**